epoxy-compose/src/main/java/com/airbnb/epoxy/ComposeInterop.kt

```diff
@@ -0,0 +1,51 @@
+package com.airbnb.epoxy
+
+import android.view.View
+import android.view.ViewGroup
+import android.widget.FrameLayout
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.ComposeView
+import androidx.compose.ui.viewinterop.AndroidView
+
+class ComposeEpoxyModel(
```

Comment:
I think we will need some sort of keying system for equals/hashcode to know when the composable changes and we need to rebind